Documentation/admin_guide.md

```diff
@@ -39,6 +39,56 @@ If `--wal-dir` flag is set, etcd will write the write ahead log files to the spe
 If you are spinning up multiple clusters for testing it is recommended that you specify a unique initial-cluster-token for the different clusters.
 This can protect you from cluster corruption in case of mis-configuration because two members started with different cluster tokens will refuse members from each other.
 
+#### Monitoring
+
+It is important to monitor your production etcd cluster for healthy information, runtime metrics, and debug variaiables.
+
+##### Health Monitoring
+
+At lowest level, etcd exposes health information via HTTP at `/health` in JSON format. If it returns `{"health": "true"}`, then the cluster is healthy. Please note the `/health` endpoint is still an experimental one as in etcd 2.2.
+
+```
+$ curl -L http://127.0.0.1:2379/health
+
+{"health": "true"}
+```
+
+You can also use etcdctl to check the cluster-wide health information. It will contact all the members of the cluster and collect the health information for you.
+
+```
+$./etcdctl cluster-health 
+member 8211f1d0f64f3269 is healthy: got healthy result from http://127.0.0.1:12379
+member 91bc3c398fb3c146 is healthy: got healthy result from http://127.0.0.1:22379
+member fd422379fda50e48 is healthy: got healthy result from http://127.0.0.1:32379
+cluster is healthy
+```
+
+##### Runtime Metrics
+
+etcd uses [Prometheus](http://prometheus.io/) for metrics reporting in the server. You can read more through the runtime metrics [doc](metrics.md).
+
+##### Debug Variables
+
+Debug variables are exposed for real-time debugging purposes. etcd exposes debug variables via HTTP at `/debug/vars` in JSON format. The debug variables contains
```

Comment:
I would say that prometheus metrics is for monitoring in a long term, while debug vars are used for debug at some certain point. Debug vars targets on develops who are familiar with this system for debugging when strange things happen, and it could be finer granuality. So I think a high-level introduction for them is good enough.

It is good to mention when to use each IMO.

> After some research it looks like there's Datadog which supports expvars, and https://github.com/divan/expvarmon.

I would not suggest that today because I don't know any one has set that successfully and it is a best practice.